src/arithmetic.jl

```diff
@@ -823,6 +890,8 @@ end
         return nothing
     end
 
+    zero!(c, k)
+
```

Comment:
I think that `zero(c, k)` makes that `div!` does not accumulate anymore the result. So I suggest to correct the note. 

Another suggestion would be to have two methods `div_accumul!` and `div` (and `mul_accumul!` and `mul!`), and use them properly. I think this would simplify `div!` below a lot...

Reply:
> I think that zero(c, k) makes that div! does not accumulate anymore the result. So I suggest to correct the note.

Good catch! Okay, I'll fix it 👍